deployments/docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: '3'
```

Comment:
I suggest to add a reference to the docker-compose format specification, so it is handy if future changes were needed.
```suggestion
# https://docs.docker.com/compose/compose-file/compose-file-v3/
version: '3'
```